lib/geom_utils.hpp

```diff
@@ -0,0 +1,147 @@
+// Copyright (c) 2010-2020, Lawrence Livermore National Security, LLC. Produced
+// at the Lawrence Livermore National Laboratory. All Rights reserved. See files
+// LICENSE and NOTICE for details. LLNL-CODE-443271.
+//
+// This file is part of the GLVis visualization tool and library. For more
+// information and source code availability see https://glvis.org.
+//
+// GLVis is free software; you can redistribute it and/or modify it under the
+// terms of the BSD-3 license. We welcome feedback and contributions, see file
+// CONTRIBUTING.md for details.
+
+#ifndef GLVIS_GEOM_UTILS_HPP
+#define GLVIS_GEOM_UTILS_HPP
+#include "mfem.hpp"
+
+// Some inline functions
+
+inline void LinearCombination(const double a, const double x[],
+                              const double b, const double y[], double z[])
```

Comment:
Were all the functions in this file just moved (without changes) from other files?

Reply:
Yes, these are just a straight copy-and-paste.